docs/application/dotnet/guides/nui/flexcontainer.md

```diff
@@ -365,34 +385,40 @@ You can modify the flex item placement through the following properties:
 
     To create the item alignment shown in the above figure:
 
-    ```
+    ```cs
     /// Create the flex container
     FlexContainer flexContainer = new FlexContainer();
 
-    /// Lay out the items horizontally
-    flexContainer.FlexDirection = FlexContainer.FlexDirectionType.Row;
-
     /// Set the default alignment along the cross axis to the beginning of the container
     flexContainer.AlignItems = FlexContainer.Alignment.AlignFlexStart;
+    flexContainer.WidthResizePolicy = ResizePolicyType.FillToParent;
+    flexContainer.HeightResizePolicy = ResizePolicyType.FillToParent;
+    flexContainer.FlexDirection = FlexContainer.FlexDirectionType.Row;
+    Window.Instance.GetDefaultLayer().Add(flexContainer);
 
     /// Create flex items and add them to the flex container
     View item1 = new View();
+
     /// Align item1 to the center of the container
-    item1.AlignSelf = FlexContainer.Alignment.AlignCenter;
+    item1.AlignSelf = (int)FlexContainer.Alignment.AlignCenter;
+    item1.Size2D = new Size2D(200, 200);
+    item1.BackgroundColor = Color.Red;
     flexContainer.Add(item1);
 
     View item2 = new View();
+
     /// Align item2 to the beginning of the container based on the default alignment
+    item2.Size2D = new Size2D(200, 200);
+    item2.BackgroundColor = Color.Green;
     flexContainer.Add(item2);
 
     View item3 = new View();
-    /// Align item3 to the bottom of the container
-    item3.AlignSelf = FlexContainer.Alignment.AlignFlexEnd;
-    flexContainer.Add(item3);
 
-    View item4 = new View();
-    /// Align item4 to the beginning of the container based on the default alignment
-    flexContainer.Add(item4);
+    /// Align item3 to the bottom of the container
+    item3.AlignSelf = (int)FlexContainer.Alignment.AlignFlexEnd;
+    item3.Size2D = new Size2D(200, 200);
+    item3.BackgroundColor = Color.Blue;
+    flexContainer.Add(item3); 
     ```
```

Comment:
Question. According to "above figure", we have 4 items. The item4 is removed with this update and not added back. Don't we need to add item4?

Reply:
To comply with above figure we can add 4th item. This item can be similar to item2 except that we directly assign `AlignSelf` property to `FlexContainer.Alignment.AlignFlexEnd`. It will show to user, that this line is not necessery thanks to inheritance.